src/DocumentDB.ChangeFeedProcessor.UnitTests/Exceptions/LeaseLostExceptionTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Microsoft.Azure.Documents.ChangeFeedProcessor.Exceptions;
+using Microsoft.Azure.Documents.ChangeFeedProcessor.PartitionManagement;
+using Moq;
+using System;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using Xunit;
+
+namespace Microsoft.Azure.Documents.ChangeFeedProcessor.UnitTests.Exceptions
+{
+    [Trait("Category", "Gated")]
+    public class LeaseLostExceptionTests
+    {
+        [Fact]
+        public void RecommendedConstructors()
```

Comment:
nit: Following other tests naming in this file, why not add a Validate_ prefix?

Reply:
+1

Reply:
done.